pkg/util/cloudfoundry/bbscache.go

```diff
@@ -196,7 +205,6 @@ func (bc *BBSCache) start() {
 func (bc *BBSCache) readData() {
 	log.Debug("Reading data from BBS API")
 	bc.Lock()
-	bc.pollAttempts++
 	bc.Unlock()
```

Comment:
I guess there is no more need to lock/unlock the BBSCache here

Reply:
Whoops, I missed that when deleting pollAttempts!